install/runtime-payload/hack/build.sh

```diff
@@ -18,6 +19,10 @@ function setup_env_for_arch() {
 			image="docker.io/library/centos"
 			kernel_arch="x86_64"
 			;;
+		"linux/s390x") 
+			image="docker.io/library/centos"
```

Comment:
Similar to [coco/operator #149](https://github.com/confidential-containers/operator/pull/149/files), this might be `clefos`? @BbolroC could you confirm?

Reply:
Yes, there is no official support for `docker.io/library/centos` for s390x. It should be `clefos` like in [cc-operator](https://github.com/confidential-containers/operator/blob/main/install/pre-install-payload/containerd/payload.sh#L27).

Reply:
actually this runtime payload image is base on `quay.io/confidential-containers/runtime-payload-ci:kata-containers-latest`, the image fields are not be used, removed them.